samples/jenkins/scripts/prod/update_model_deployment.sh

```diff
@@ -0,0 +1,27 @@
+export PATH=$PATH:$PWD
+
+
+prod_space_id=$(cpdctl space list --output json --jmes-query "resources[?entity.name == 'prod-space'] | [0].metadata.id" --raw-output)
+echo "Production space: $prod_space_id"
+
+export CPD_SCOPE=cpd://cpd/spaces/$prod_space_id
+
+prod_model_id=$(cpdctl asset search --query '*:*' --type-name wml_model --output json \
+  --jmes-query "results[0].metadata.asset_id" --raw-output)
+echo "Model id: $prod_model_id"
+
+prod_model_rev=$(cpdctl ml model list-revisions --model-id {prod_model_id} --output json \
+  --jmes-query "resources[0].metadata.rev" --raw-output)
+echo "Model revision: $prod_model_rev"
+
+prod_model_batch_deployment_id=$(cpdctl ml deployment list --asset-id "$prod_model_id" --output json \
+  --jmes-query "resources[0].metadata.id" --raw-output)
+echo "Batch deployment id: $prod_model_batch_deployment_id"
+
+export asset='{"id": "'$prod_model_id'",
+              "rev": "'$prod_model_rev'"}'
+
+cpdctl ml deployment update --deployment-id "$prod_model_batch_deployment_id" --asset "$asset"
+
+echo 'Updated deployed model revision:'
+! cpdctl ml deployment get  --deployment-id "$prod_model_batch_deployment_id" --output json -j 'entity.asset'
```

Comment:
Same as in the very first comment about `!` mark.

Reply:
good catch!